extensions/amp-consent/0.1/test/test-consent-config.js

```diff
@@ -46,7 +48,7 @@ describes.realWin('ConsentConfig', {amp: 1}, env => {
     it('read inline config', () => {
       appendConfigScriptElement(doc, element, defaultConfig);
       const consentConfig = new ConsentConfig(element);
-      expect(consentConfig.getConsentConfig()).to.deep.equal(
+      expect(consentConfig.getConsentConfigPromise()).to.eventually.deep.equal(
```

Comment:
you need to `return` all the promise assertion.
all, you could change to `async` syntax.

```
it ('xxx', async () => {
  expect(await consentConfig.getConsentConfigPromise()).to....
});
```

BTW, you should try to fail the test first then make it pass to make sure the test is not silently passing.

Reply:
Good suggestion. Will fail them first :)